pkg/plugins/models.go

```diff
@@ -207,6 +207,8 @@ type PluginMetaDTO struct {
 
 	Module  string `json:"module"`
 	BaseURL string `json:"baseUrl"`
+
+	CachingConfig EnterpriseCachingConfig `json:"cachingConfig,omitempty"`
```

Comment:
Nit. Maybe `QueryCachingConfig` is a better name?

Reply:
done [a9877c6](https://github.com/grafana/grafana/pull/61968/commits/a9877c60483ea51669d429c9e882933da0903ae7)